internal/providers/packet/packet.go

```diff
@@ -69,6 +69,7 @@ func FetchMetadata() (providers.Metadata, error) {
 
 func parseNetwork(network metadata.NetworkInfo) ([]providers.NetworkInterface, error) {
 	ifaces := []providers.NetworkInterface{}
+	bondDev := "bond0"
```

Comment:
Minuscule nit: why not just do `Name: "bond"` further down? Making an extra variable for this seems unnecessary.